Runtime/Behaviours/ZoomDhvCamera.cs

```diff
@@ -0,0 +1,80 @@
+using UnityEngine;
+using ReupVirtualTwin.inputs;
+using Unity.Cinemachine;
+using ReupVirtualTwin.managers;
```

Comment:
I just left a comment below why I think this should be like this. Spoiler: it's to make it harder for us to make a mess in the future
```suggestion
using ReupVirtualTwin.managerInterfaces;
```